lxd/db/cluster/config.mapper.go

```diff
@@ -89,8 +89,9 @@ func GetConfig(ctx context.Context, tx *sql.Tx, parent string, filters ...Config
 
 	configObjectsLocal := strings.Replace(configObjects, "%s_id", fmt.Sprintf("%s_id", parent), -1)
 	fillParent := make([]any, strings.Count(configObjectsLocal, "%s"))
+	mangledParent := strings.Replace(parent, "_", "s_", -1) + "s"
 	for i := range fillParent {
-		fillParent[i] = strings.Replace(parent, "_", "s_", -1) + "s"
+		fillParent[i] = mangledParent
```

Comment:
FYI These files are generated by `lxd-generate`. You'll need to fix `lxd-generate` to get it to output what you want here.

Reply:
Oh, nice thanks!

I tried to run `make update-schema` but it seems like I'm missing the `~/go/src` dir:

```
$ ll ~/go/
total 125
drwxrwxr-x  5 sdeziel sdeziel  5 Nov 11 16:09 ./
drwx------ 47 sdeziel sdeziel 69 Dec 18 14:13 ../
drwxrwxr-x  3 sdeziel sdeziel 29 Dec 18 14:15 bin/
drwxrwxr-x  3 sdeziel sdeziel  3 Dec  3 20:32 deps/
drwxrwxr-x  4 sdeziel sdeziel  4 Dec  4 10:49 pkg/
```

```
$ make update-schema 
cd lxd/db/generate && go build -v -trimpath -o /home/sdeziel/go/bin/lxd-generate -tags "libsqlite3"  && cd -
/home/sdeziel/git/lxd
go generate ./...
Error: Update cluster database schema: failed to open Go file for writing: open /home/sdeziel/go/src/github.com/canonical/lxd/lxd/db/cluster/schema.go: no such file or directory
Usage:
  lxd-generate db schema [flags]

Flags:
  -h, --help   help for schema

lxd/db/schema.go:7: running "lxd-generate": exit status 1
make: *** [Makefile:147: update-schema] Error 1
```

So noob question: how to I get it to work?

Reply:
This seems to be an accidental regression due to https://github.com/canonical/lxd/commit/63b017e84dab2082af8a84b81238b50a82e4e0a3

Putting a symlink into place to have `~/go/src/github.com/canonical/lxd` actually exist works.